Library/Homebrew/test/spec_helper.rb

```diff
@@ -177,7 +177,7 @@ def find_files
       @__stdout = $stdout.clone
       @__stderr = $stderr.clone
 
-      unless example.metadata.key?(:focus) || ENV.key?("VERBOSE_TESTS")
+      unless (example.metadata.keys & [:focus, :byebug]).any? || ENV.key?("VERBOSE_TESTS")
```

Comment:
```suggestion
      if (example.metadata.keys & [:focus, :byebug]).empty? || !ENV.key?("VERBOSE_TESTS")
```

Reply:
What would you think about just using `focus` here?

Reply:
That can be done but I have 2 points in mind:`focus` restricts which examples are being run.

Reply:
This can be done, but I think we should consider the following 2 points:
* `focus` restricts which example/spec is being run, hence, if we want to debug a single spec and run all the others, `focus` would not let that happen.
* Using byebug with focus (a totally different rspec feature) may cause ambiguity.

Reply:
I'll look more at the test code for this once the vendored files are gone, a bit hard to read for now!